xlml/utils/xpk.py

```diff
@@ -63,19 +64,30 @@ def run_workload(
       multi_keyword = "num-nodes"
     else:
       multi_keyword = "num-slices"
-    cmds = (
-        "set -xue",
-        f"git clone https://github.com/google/xpk {tmpdir}/xpk",
-        (
-            f"python {tmpdir}/xpk/xpk.py workload create"
-            f" --cluster={cluster_name} --workload={workload_id}"
-            f" --command='{run_cmds}' --device-type={accelerator_type}"
-            f" --{multi_keyword}={num_slices} --docker-image={docker_image}"
-            f" --project={cluster_project} --zone={zone}"
-            f" --env {metric_config.SshEnvVars.GCS_OUTPUT.name}={gcs_path}"
-            " --restart-on-user-code-failure"
-        ),
+
+    workload_create_cmd = (
+        f"python {tmpdir}/xpk/xpk.py workload create"
+        f" --cluster={cluster_name} --workload={workload_id}"
+        f" --command='{run_cmds}' --device-type={accelerator_type}"
+        f" --{multi_keyword}={num_slices} --docker-image={docker_image}"
+        f" --project={cluster_project} --zone={zone}"
+        f" --env {metric_config.SshEnvVars.GCS_OUTPUT.name}={gcs_path}"
+        " --restart-on-user-code-failure"
     )
+    if use_vertex_tensorboard:
+      workload_create_cmd += " --use-vertex-tensorboard"
+      cmds = (
+          "set -xue",
+          f"git clone https://github.com/google/xpk {tmpdir}/xpk",
+          "pip install -U google-cloud-aiplatform cloud-accelerator-diagnostics",
+          workload_create_cmd,
+      )
+    else:
+      cmds = (
+          "set -xue",
+          f"git clone https://github.com/google/xpk {tmpdir}/xpk",
+          workload_create_cmd,
+      )
```

Comment:
Could we rewrite something like this? So we have slightly concise code block.

```
vertext_dependency = "pip install -U google-cloud-aiplatform cloud-accelerator-diagnostics" if use_vertex_tensorboard else None

      cmds = (
          "set -xue",
          f"git clone https://github.com/google/xpk {tmpdir}/xpk",
          vertext_dependency,
          workload_create_cmd,
      )
```

Reply:
This will not work for tests where vertex_dependency="". The error will be thrown as https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/xlml/utils/xpk.py#L81 would result in `git clone https://github/com/google/xpk tmp54/xpk;;python...`. I will update `cmds` to a list instead of tuple to improve the readability of the code.

Reply:
Ah, I see. Makes sense. Thanks!